pkg/api/addons.go

```diff
@@ -32,16 +32,16 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 	}
 
 	defaultTillerAddonsConfig := KubernetesAddon{
-		Name:    DefaultTillerAddonName,
+		Name:    TillerAddonName,
```

Comment:
These addon name consts are literally static consts, and not overridable, so the `Default` prefix is unnecessary (and misleading)